src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.cpp

```diff
@@ -265,6 +267,10 @@ namespace OpenMS
     {
       // Dimethyl or ICPL
       // We assume each sample to be labelled only once. Hence, we only consider samples_labels_[...][0] below.
+      // Unlike in classical SILAC where two labels with two different specificities are available, in Dimethyl labelling
+      // there is only one specificity (Lys and N-term). Two labels [Lys8,Arg10] in a SILAC sample are fine, since
+      // both have different specificities. But two labels [Dimethyl4,Dimethyl8] make no sense, since both have the same
+      // specificity. With only one specificity in Dimethyl labelling available, each sample can have only one label.
```

Comment:
aren't these two different things? Lys and any N-term?
Maybe I don't understand how the term specificity is used?